tests/integration_tests/functional/test_cpu_features.py

```diff
@@ -572,6 +572,32 @@ def _test_cpu_wrmsr_restore(context):
     microvm_factory = context.custom["microvm_factory"]
```

Comment:
nit: commit message: I think we should mention in the subject that this fix is for a test, preferably pointing to a specific test (or file).

Reply:
I have changed it to "fix:test_cpu_features:skip restore if artifacts not found", let me know if it sounds ok

Reply:
nit: if you can fit spaces after colons, would be perfect